### ISSUE TITLE
[Issue]: Slow overall execution when generating

### ISSUE THREAD

**Issue by Tillerz** [vladmandic/automatic#3604]
### Issue Description

Generally slow execution of several steps of the rendering process, here same examples (excerpt from the full log below):

```
2024-11-29 13:41:14,915 | sd | INFO | processing | Processed: images=1 its=3.19 time=3.13 timers={'init': 1.14, 'prepare': 0.05, 'move': 0.57, 'encode': 5.13, 'preview': 1.37, 'pipeline': 3.57, 'hires': 8.14, 'decode': 3.46, 'post': 0.03} memory={'ram': {'used': 4.97, 'total': 39.18}, 'gpu': {'used': 2.12, 'total': 23.99}, 'retries': 0, 'oom': 0}
2024-11-29 13:41:14,927 | sd | INFO | processing | Processed: images=1 its=0.60 time=16.80 timers={'init': 1.14, 'prepare': 0.05, 'move': 0.57, 'encode': 5.13, 'preview': 1.37, 'pipeline': 3.57, 'hires': 8.14, 'decode': 3.46, 'post': 0.04} memory={'ram': {'used': 4.97, 'total': 39.18}, 'gpu': {'used': 2.12, 'total': 23.99}, 'retries': 0, 'oom': 0}
2024-11-29 13:43:04,119 | sd | INFO | processing | Processed: images=1 its=0.84 time=11.87 timers={'move': 0.19, 'encode': 4.07, 'pipeline': 1.53, 'preview': 0.19, 'hires': 7.16, 'decode': 2.18, 'post': 0.87} memory={'ram': {'used': 5.06, 'total': 39.18}, 'gpu': {'used': 1.78, 'total': 23.99}, 'retries': 0, 'oom': 0}
2024-11-29 13:43:26,270 | sd | INFO | processing | Processed: images=1 its=3.30 time=3.03 timers={'init': 0.71, 'prepare': 0.04, 'move': 0.41, 'encode': 4.69, 'pipeline': 3.05, 'preview': 0.85, 'hires': 7.69, 'decode': 3.36, 'post': 0.03} memory={'ram': {'used': 5.07, 'total': 39.18}, 'gpu': {'used': 2.13, 'total': 23.99}, 'retries': 0, 'oom': 0}
2024-11-29 13:43:26,282 | sd | INFO | processing | Processed: images=1 its=0.66 time=15.17 timers={'gc': 0.02, 'init': 0.71, 'prepare': 0.04, 'move': 0.41, 'encode': 4.69, 'pipeline': 3.05, 'preview': 0.85, 'hires': 7.69, 'decode': 3.36, 'post': 0.04} memory={'ram': {'used': 5.07, 'total': 39.18}, 'gpu': {'used': 2.13, 'total': 23.99}, 'retries': 0, 'oom': 0}
2024-11-29 13:43:36,080 | sd | INFO | processing | Processed: images=1 its=2.58 time=3.88 timers={'init': 0.03, 'move': 0.19, 'encode': 1.28, 'preview': 0.52, 'pipeline': 1.05, 'hires': 2.18, 'decode': 0.43, 'post': 0.08} memory={'ram': {'used': 5.08, 'total': 39.18}, 'gpu': {'used': 12.69, 'total': 23.99}, 'retries': 0, 'oom': 0}
```

I tried some additional things (not part of the log):
- Disabled text encoder (set to default none), and switched from Hinker to Native: no improvement.
- Disabled PAG and HiDiffusion: no improvement:

```
Time: 7.51s | init 0.16 move 0.37 encode 1.44 pipeline 2.06 preview 0.34 hires 2.24 decode 1.70 | GPU 11288 MB 9%
Time: 15.24s | init 0.70 move 0.38 encode 4.49 pipeline 2.48 preview 0.86 hires 7.36 decode 3.36 | GPU 15104 MB 9%
Time: 5.25s | init 0.15 move 0.36 encode 1.41 pipeline 0.86 hires 2.19 decode 0.66 | GPU 11102 MB 52%
Time: 15.04s | init 0.72 move 0.40 encode 4.21 pipeline 2.52 preview 0.45 hires 7.04 decode 3.36 | GPU 15100 MB 9%
```

### Version Platform Description

```
Win11, WSL-2, RTX 4090, b74166f9
Processor	AMD Ryzen 7 7700X 8-Core Processor                4.50 GHz
Installed RAM	64,0 GB (63,2 GB usable)
drives are all SSD
```

### Relevant log output

see attachment

### Backend

Diffusers

### UI

Standard

### Branch

Master

### Model

StableDiffusion 1.5

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by Tillerz** [comment]
[sdnext.log](https://github.com/user-attachments/files/17959536/sdnext.log)

**Comment by brknsoul** [comment]
I believe this is due to Live Preview slowing things down.

I'll attached the gifs I took;

Live preview display period = 1, method == approximate
![chrome_Tyx3p6RXio](https://github.com/user-attachments/assets/a35a596d-6026-473a-a7d0-1eea8622ee4e)

Live preview display period = 0 (ie: disabled);
![chrome_fm3XBE8ce5](https://github.com/user-attachments/assets/4a04561e-c666-43b6-a873-34eefda2ce71)

**Comment by Tillerz** [comment]
Ok, I checked up and down... I think I found the problem. GPUTweak wasn't set to Default but User, and that was limited to 80% power and max 84°C. Setting it to default results in this. I think that looks way leathier for an RTX 4090. I am glad it's not a weird issue with SDNext.

Processed: images=1 its=1.87 time=5.35 timers={'gc': 0.03, 'init': 0.19, 'prepare': 0.04, 'move': 0.7, 'encode': 1.68, 'preview': 0.12, 'pipeline': 1.27, 'hires': 2.21, 'decode': 0.93, 'post': 0.18}
